web/packages/extension/src/background.ts

```diff
@@ -22,7 +22,7 @@ function isSwf(
         .match(/^\s*(.*?)\s*(?:;.*)?$/)![1];
 
     // Some sites (e.g. swfchan.net) might (wrongly?) send octet-stream, so check file extension too.
-    if (mime === "application/octet-stream") {
+    if (mime.endsWith("octet-stream")) {
```

Comment:
Maybe we should just allow specifically `application/octet-stream` and `binary/octet-stream`? Tough I don't have any good argument for/againt this change...

Reply:
What's weird is there is only supposed to be one type of octet-stream form my understanding: https://docs.w3cub.com/http/basics_of_http/mime_types/complete_list_of_mime_types.html

I don't really know where binary/octet-stream comes from.

Reply:
Comes from AWS: probably shouldn't be a thing but impossible to change now due to backwards compatibility: https://github.com/aws/aws-sdk-java/issues/1143#issuecomment-298739373

Reply:
https://github.com/aws/aws-sdk-java/issues/1143#issuecomment-1037051062:

> Internet lore and my personal failing memory has it that some old browsers (like IE) wanted to save anything with type 'application/octet-stream' as a .zip file, and a common way to avoid this was to use the unknown type 'binary/octet-stream'.